src/Sylius/Component/Resource/spec/Exception/UnexpectedTypeExceptionSpec.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace spec\Sylius\Component\Resource\Exception;
+
+use PhpSpec\ObjectBehavior;
+
+class UnexpectedTypeExceptionSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith('givenType', 'expectedType');
```

Comment:
Maybe change `givenType` to `stringValue` to indicate it's string object and `expectedType` to `\ExpectedType` to indicate it's string class name, since it will be more intuitive.